src/modules/dashboard/i18n/en_AU.root.json

```diff
@@ -0,0 +1,3 @@
+{
+    "name": "Dashboard"
+}
```

Comment:
Is it just me or is the en_GB.root file missing?

Reply:
is here https://github.com/LSS-Manager/LSSM-V.4/blob/en_GB/src/modules/dashboard/i18n/en_GB.root.json